docs/install/install-knative-gcp.md

```diff
@@ -14,180 +16,67 @@
-      projects.
-
-      ```shell
-      gcloud projects add-iam-policy-binding $PROJECT_ID \
-        --member=serviceAccount:cloud-run-events@$PROJECT_ID.iam.gserviceaccount.com \
-        --role roles/owner
-      ```
-
-      Then, you can either:
-
-      - Use **Workload Identity**.
+## Install the Knative-GCP Constructs
+
+### Option 1: Install from Master using [ko](http://github.com/google/ko)
+
+```shell
+ko apply -f ./config
+```
+
+### Option 2: Install a [release](https://github.com/google/knative-gcp/releases).
+
+Remember to update `{{< version >}}` in the commands below with the
+appropriate release version.
+
+1. First install the CRDs by running the `kubectl apply` command with the
+   `--selector` flag. This prevents race conditions during the install,
+   which cause intermittent errors:
+
+   ```shell
+   kubectl apply --selector messaging.cloud.google.com/crd-install=true \
+   --filename https://github.com/google/knative-gcp/releases/download/{{< version >}}/cloud-run-events.yaml
+   kubectl apply --selector events.cloud.google.com/crd-install=true \
+   --filename https://github.com/google/knative-gcp/releases/download/{{< version >}}/cloud-run-events.yaml
+   ```
+
+2. To complete the install run the `kubectl apply` command again, this time
+   without the `--selector` flags:
+
+   ```shell
+   kubectl apply --filename https://github.com/google/knative-gcp/releases/download/{{< version >}}/cloud-run-events.yaml
+   ```
+
+## Configure the Authentication Mechanism for GCP
 
-        Workload Identity is the recommended way to access Google Cloud services
-        from within GKE due to its improved security properties and
-        manageability. For more information about Workload Identity, please see
-        [here](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+Currently, we support two methods: Workload Identity and Kubernetes Secret.
+Workload Identity is the recommended way to access Google Cloud services from
+within GKE due to its improved security properties and manageability. For more
+information about Workload Identity, please see [here](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
 
-        1. Enable Workload Identity.
+**_Note_**: Before applying initialization scripts, make sure your default zone is set to be
+the same as your current cluster. You may use
+`gcloud container clusters describe $CLUSTER_NAME` to get zone and apply
+`gcloud config set compute/zone $ZONE` to set it.
 
-           ```shell
-           gcloud services enable iamcredentials.googleapis.com
-           gcloud beta container clusters update $CLUSTER_NAME \
-           --identity-namespace=$PROJECT_ID.svc.id.goog
-           ```
+**_Note_**: Both scripts will have a step to create a Google Cloud Service
+Account `cloud-run-events`. Ignore the error message if you already had
+this service account (error for 'service account already exists').
```

Comment:
Is it worth altering the scripts so that users don't see an error? Not in this PR, but maybe as a backlog task (with a TODO here).

Reply:
Added a TODO and created an issue: https://github.com/google/knative-gcp/issues/896